### ISSUE TITLE
UI Scaling Issues

### ISSUE THREAD

**Issue by VenteF** [TeamStarDrive/StarDrive#151]
**Description**
I'm experiencing issues with the game's UI scaling on my screen with a resolution of 1366x768. The skill tree icons overlap, and the buttons on the top of the main interface (specifically "Ships Array" and "Fleets") are missing.
**Steps to reproduce**
Launch the game on a screen with a resolution of 1366x768.
Observe the skill tree icons and top interface buttons.
**Expected behavior**
The UI should scale correctly to fit the screen resolution, with all elements visible and properly aligned.
**Actual behavior**
The skill tree icons overlap, and the top interface buttons ("Ships Array" and "Fleets") are missing.
**Investigation so far**
I've found some UI settings in stardriveplus/content/ui (YAML files), but I'm unsure what to tweak to fix the scaling for my screen size. The issue is present in Black box as well when you add Combined Arms mod.
**Additional information**
Screen resolution: 1366x768
Game version: Blackbox Mars 1.50.15014 & CombinedArms 8.5C 
Operating System: Windows 10 Pro 22h2 - Windows Feature Experience Pack 120.2212.4170.0
**Request**
Please assist in resolving this issue or provide guidance on how to adjust the UI settings to fix the scaling for my screen size. 

Thank you!

**Comment by rxnnxs** [comment]
You are right with this observation and even on a resolution 1920x1080 there are things that are too broad.
The thing is, that Ships Array (K) and Fleets (J) is pushed aside. Thats what I assume..
You can open them with the key-shortcuts k and j.

You could try to shorten the text in gameText.yaml

You search for 
ShipsArray:
 Id: 104
 ENG: "Ships Array"
 
 and change it to "Ships"
 
 Do for the rest in the top buttons so that they all can fit.
 Please report back and make some screenshots to show us where else it is happening.
